.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,3 +2,19 @@
 **Asana Ticket:** [TICKET_NAME](TICKET_LINK)
 
 [Please include a brief description of what was changed]
+
+---
+
+Before getting review, please check the following:
+
+* [ ] Does frontend functionality render and work correctly in IE?
+* [ ] Has every endpoint that will be polled from the frontend been load-tested?
+* [ ] Do form elements have ARIA labels?
```

Comment:
I'd suggest expanding this to "Are interactive elements accessible to screen readers?" In most cases, making elements accessible will simply involve using appropriate semantic markup (for example `<label>` tags wrapping a form control), rather than adding ARIA attributes.